doc/btp_mesh_node.txt

```diff
@@ -226,6 +226,8 @@ Commands and responses:
 		Command parameters:     Network index (2 octets)
 		                        Node address (2 octets)
 		                        Page (1 octet)
+		Response parameters:	Received Page (1 octet)
+								Composition Data (variable length)
```

Comment:
Extra tabs?

Reply:
Same number of tabs; I am not sure why it looks weird here. 
![image](https://github.com/auto-pts/auto-pts/assets/94102255/4b8fec23-8e8e-4d70-bb2c-0736bc783f94)

Reply:
changed the tabs with space as the rest of the file has